pkg/apis/pipeline/v1alpha1/taskrun_validation.go

```diff
@@ -24,13 +24,15 @@ import (
 	"k8s.io/apimachinery/pkg/api/equality"
 )
 
+// Validate taskrun
 func (t *TaskRun) Validate() *apis.FieldError {
 	if err := validateObjectMetadata(t.GetObjectMeta()).ViaField("metadata"); err != nil {
 		return err
 	}
 	return t.Spec.Validate()
 }
 
+// Validate taskrun spec
```

Comment:
i know the linters want this, but if we're going to include a comment here, can we add some more information? maybe one or both of:

- Talk about how this is going to be used (called by the controller on creation?)
- Talk about what it means for the TaskRun to be valid

Reply:
going to leave as is for now, will refactor PipelineRun and TaskRun comments when the refactor goes in combining their logic